# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PContent.ipdl
# Commit: 6450088b6b73
# Full Hash: 6450088b6b73ffc17c79cd7097b1bfe30d00e207
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2020-09-14 21:29:41
# Regressor Bug: 1627075
# File Overlap Count: 1
# Description:
#   Bug 1656261 - Back out all recent StartupCache work r=RyanVM
#   
#   This backs out all work from bug 1627075 as well as all of its
#   descendents. There were a few conflicts when backing this out but
#   overall it was pretty clean, so I would say it's a fairly mild
# ==============================================================================

diff -r b0fabcd60430 -r 6450088b6b73 dom/ipc/PContent.ipdl
--- a/dom/ipc/PContent.ipdl	Mon Sep 14 17:32:40 2020 +0000
+++ b/dom/ipc/PContent.ipdl	Mon Sep 14 17:00:53 2020 +0000
@@ -44,7 +44,6 @@
 include protocol PRemoteDecoderManager;
 include protocol PProfiler;
 include protocol PScriptCache;
-include protocol PStartupCache;
 include protocol PSessionStorageObserver;
 include protocol PBenchmarkStorage;
 include DOMTypes;
@@ -416,7 +415,6 @@
     manages PURLClassifier;
     manages PURLClassifierLocal;
     manages PScriptCache;
-    manages PStartupCache;
     manages PLoginReputation;
     manages PSessionStorageObserver;
     manages PBenchmarkStorage;
@@ -546,8 +544,6 @@
 
     async PScriptCache(FileDescOrError cacheFile, bool wantCacheData);
 
-    async PStartupCache();
-
     async RegisterChrome(ChromePackage[] packages, SubstitutionMapping[] substitutions,
                          OverrideMapping[] overrides, nsCString locale, bool reset);
     async RegisterChromeItem(ChromeRegistryItem item);